code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4855,3 +4855,15 @@
 	..()
 	reagents.add_reagent(NUTRIMENT, 6)
 	bitesize = 2
+
+/obj/item/weapon/reagent_containers/food/snacks/poutine
+	name = "Poutine"
```

Comment:
You should remove the capitalization here, because BYOND will treat it as a proper noun and examine text and such will say "That's Poutine"